scripts/deploy.sh

```diff
@@ -0,0 +1,74 @@
+export DEBIAN_FRONTEND=noninteractive
+
+# Install build dependencies
+echo "deb http://ftp.de.debian.org/debian sid main" >> /etc/apt/sources.list
+apt update -y
+apt install git curl ufw gcc make nginx -y
+apt install gcc-10 -y
+
+# Install MeiliSearch v0.10.0
+wget --directory-prefix=/etc/meilisearch/ https://github.com/meilisearch/MeiliSearch/releases/download/v0.10.0/meilisearch.deb
+apt install /etc/meilisearch/meilisearch.deb
+
+# Prepare systemd service for MeiliSearch
+cat << EOF >/etc/systemd/system/meilisearch.service
+[Unit]
+Description=MeiliSearch
+After=systend-user-sessions.service
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/meilisearch
+
+[Install]
+WantedBy=default.target
+EOF
+
+# Start MeiliSearch service
+systemctl enable meilisearch
+systemctl start meilisearch
+
+# Delete default Nginx config
+rm /etc/nginx/sites-enabled/default
+
+# Set Nginx to proxy MeiliSearch
+cat << EOF > /etc/nginx/sites-enabled/default
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+
+    server_name _;
+
+    location / {
+        proxy_pass  http://127.0.0.1:7700;
+    }
+}
+EOF
+systemctl restart nginx
+
+# Clean up image using DigitalOcean scripts
+curl https://raw.githubusercontent.com/digitalocean/marketplace-partners/master/scripts/cleanup.sh | bash
+
+# Reset os-release file
```

Comment:
Why should we reset the os-release file? Is it modified?

Reply:
I am still trying to figure out why, but the **DO cleaning script** modifies it and then DO can't recongnize the version of the OS, so they reject the image. I'll try to find why is that happening but if it is too time demanding, I'll just copy the file before running the script, then put it back, cause it is ugly of course. Just a temporary fix to be able to build the image. I'll make an issue to avoid forgetting it

Reply:
Created #7